app/scenes/Settings/Slack.js

```diff
@@ -17,11 +17,11 @@ class Slack extends Component {
         <h1>Slack</h1>
         <HelpText>
           Connect Outline to your Slack team to instantly search for documents
-          using the <Code>/outline</Code> command.
+          using the <Code>/outline</Code> command and unfurl Outline links.
```

Comment:
I think "preview Outline links" would be more descriptive than "unfurl"

Reply:
agree